integration_tests/pages/v2Manage/outOfServiceBeds/outOfServiceBedShow.ts

```diff
@@ -71,6 +71,10 @@ export class OutOfServiceBedShowPage extends Page {
     })
   }
 
+  shouldShowUpdateConfirmationMessage() {
+    this.shouldShowBanner('The out of service bed has been updated')
```

Comment:
👍 I see this matches what we have in the 'create' action: https://github.com/ministryofjustice/hmpps-approved-premises-ui/blob/main/server/controllers/v2Manage/outOfServiceBedsController.test.ts#L95-L125

To my mind this does read better than "The out of service bed _record_ has been updated" but we can see what others think...